packages/wallets/src/features/cfd/screens/Onfido/Onfido.scss

```
@@ -1,25 +1,90 @@
 .wallets-onfido {
     width: 99.6rem;
-    min-height: 47.2rem;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
     padding: 1.6rem 0;
-    overflow-y: auto;
+
+    @include desktop {
+        max-height: calc(var(--wallets-vh, 1vh) * 100 - 23.5rem);
+        overflow-y: scroll;
+    }
 
     @include mobile {
-        width: auto;
-        height: 100%;
+        width: 100%;
         padding: 1.6rem;
+    }
 
-        & > :first-child {
+    &__content {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        @include mobile {
+            min-height: 47.2rem;
+        }
+    }
+
+    &__wrapper {
+        position: relative;
+        min-height: 38.2rem;
+        min-width: 51.2rem;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        transition: all 1s ease;
+
+        @include mobile {
+            width: 100%;
+            min-width: auto;
+        }
+
+        &-onfido-container {
+            width: 100%;
             display: flex;
-            justify-content: center;
+            flex-direction: column;
+            align-items: center;
         }
 
         // stylelint-disable-next-line selector-class-pattern
         & .onfido-sdk-ui-Modal-inner {
-            margin-inline: 3.9rem;
-            width: 32.8rem;
-            min-height: 38.2rem;
-            height: 60%; // height from figma is too small so make it relative
+            border-radius: 8px;
+            min-height: 62rem;
+        }
+
+        &-overlay {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            transition: all 0.2s ease;
+            background-color: rgba(#fff, 0.76);
```

is there a similar css value for this maybe try checking in themes. Or better yet what you can do is:

```suggestion
            background-color:[ rgba(#fff, 0.76)](background: var(--general-main-2, #ffffff));
            filter: opacity(0.76)
```
This way you don't have to go through the Themes.css at core and also use the css variables. 👍🏼